client/my-sites/tasks/show/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+import { getSelectedSiteId } from 'state/ui/selectors';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+
+class Show extends PureComponent {
+	render() {
+		const { siteId } = this.props;
+
+		return (
+			<p>
+				{ siteId && <QuerySiteChecklist siteId={ siteId } /> }
+				{ this.props.translate( 'show!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **105** times:
`translate( 'Show' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tasks-section)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).